### PR TITLE
Class 5 & 6 PR request.

### PR DESCRIPTION
**Pull request by fahimar**

### CODE REVIEW

File: class-2/Problem-1.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+// Mark and John are trying to compare their BMI (Body Mass Index), which is
```
Comment (tajnur007) [inline class-2/Problem-1.js:1]: This problem solution looks like copy paste. As far I can see you normally do not declare variables this way. Isn't it copy-paste?

File: class-2/Problem-3.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+// There are two gymnastics teams, Dolphins and Koalas. They compete against each
```
Comment (tajnur007) [inline class-2/Problem-3.js:1]: Same for this file :(

File: class-2/Problem-1.js (hunk 1 above)
Comment (fahimar) [reply class-2/Problem-1.js:1]: The question is copy pasted but the solution was completely by mine. As I thought, later the q.s for this solution will make sense when I turn on the file that's why it is copy pasted.

File: class-2/Problem-3.js (hunk 2 above)
Comment (fahimar) [reply class-2/Problem-3.js:1]: The question is copy pasted but the solution was completely by mine.

File: class-2/Problem-1.js (hunk 1 above)
Comment (fahimar) [reply class-2/Problem-1.js:1]: And yes I tried const variable first time.

**Review by tajnur007 — APPROVED** [review]
I'm approving this PR but I think you should do more practice on class 4 & 6. Also I had put some comment and hope that you will also keep focus on those. Thanks!

PR Approved!

**Pull request merged**